### PR TITLE
Add sodium 0.6.0.

### PR DESCRIPTION
**Pull request by yallop**
/cc @dsheets

### CODE REVIEW

File: packages/sodium/sodium.0.6.0/opam [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+bug-reports: "https://github.com/dsheets/ocaml-sodium/issues/"
+tags: "org:mirage"
+dev-repo: "https://github.com/dsheets/ocaml-sodium.git"
+build: [
+  [make] { os != "freebsd" }
+  [make "CFLAGS=-I/usr/local/include -L/usr/local/lib"] { os = "freebsd" }
+]
+install: [
+  [make "PREFIX=%{prefix}%" "install"]
+]
```
Comment (hcarty) [inline packages/sodium/sodium.0.6.0/opam]: Duplicate install fields

Comment (yallop) [inline packages/sodium/sodium.0.6.0/opam]: Thanks! Now fixed.

**Comment by avsm** [comment]
```
#=== ERROR while installing sodium.0.6.0 ======================================#
# opam-version         1.2.2 (58ef3b8213100953848d362f7120a30356d7f77d)
# os                   linux
# command              make
# path                 /home/opam/.opam/4.03.0/build/sodium.0.6.0
# compiler             4.03.0
# exit-code            2
# env-file             /home/opam/.opam/4.03.0/build/sodium.0.6.0/sodium-29-736596.env
# stdout-file          /home/opam/.opam/4.03.0/build/sodium.0.6.0/sodium-29-736596.out
# stderr-file          /home/opam/.opam/4.03.0/build/sodium.0.6.0/sodium-29-736596.err
### stdout ###
# [...]
# lib/sodium_stubs.c:646:4: error: implicit declaration of function 'crypto_pwhash_argon2i_str' [-Werror=implicit-function-declaration]
#     int x490 = crypto_pwhash_argon2i_str(x479, x480, x481, x484, x487);
#     ^
# lib/sodium_stubs.c: In function 'caml__75_crypto_pwhash_argon2i_str_verify':
# lib/sodium_stubs.c:655:4: error: implicit declaration of function 'crypto_pwhash_argon2i_str_verify' [-Werror=implicit-function-declaration]
#     int x499 = crypto_pwhash_argon2i_str_verify(x494, x495, x496);
#     ^
# cc1: all warnings being treated as errors
# Command exited with code 2.
# Makefile:13: recipe for target 'all' failed
### stderr ###
# make: *** [all] Error 10
```

**Comment by yallop** [comment]
It's a depext version error: `ocaml-sodium` requires at least version 1.0.9 of libsodium, but the various CI jobs mostly seem to install older versions.

**Pull request merged**